api/Gemfile

```diff
@@ -1,6 +1,7 @@
 eval(File.read(File.dirname(__FILE__) + '/../common_mnoe_dependencies.rb'))
 
 gem 'mno-enterprise-core', path: '../core'
+gem 'mno-enterprise-frontend', path: '../frontend'
```

Comment:
The Travis build was failing because the spec in `api` can't find the layout in `frontend`, so had to include `frontend` gem in `api`.